src/app.jsx

```diff
@@ -314,6 +314,7 @@ class Application extends React.Component {
                                 delete copyStats[reply.Id + system.toString()];
                                 return { containersDetails: copyDetails, containersStats: copyStats };
                             });
+                            this.inspectContainerDetail(reply.Id, system);
```

Comment:
I'd like to put a hold on this. I am currently completely reworking the state/API handling to not call inspectContainerDetail() and getContainers() a zillion times (it causes flakes and makes everything slow), see #1324. I should be done today, then we can rebase this PR and avoid this call.

Reply:
Sure no problem

Reply:
Unblocked. This call can be dropped now.